### ISSUE TITLE
Error baking when using tags in a post

### ISSUE THREAD

**Issue by kfowlks** [ludovicchabant/PieCrust#144]
<p>
Hello, 
It appears I've run into an issue when attempting to post a blog with tags in Piecrust chef version 1.3.0-dev.

<code>
[fowlks@node1a test]$ chef bake
[     1.8 ms] cleaned cache (reason: need bake record regen)
[    99.9 ms] 2014/06/13/check_me
[     2.6 ms] 2014/06/16/next_level
PHP Fatal error:  Call to undefined function PieCrust\Util\mb_strtolower() in /home2/PieCrust/src/PieCrust/Util/UriBuilder.php on line 177
Critical error in '/home2/PieCrust/src/PieCrust/Util/UriBuilder.php:177': Call to undefined function PieCrust\Util\mb_strtolower()
</code>

When running this the chef tags command it appears that the tags get recognized properly.  
<code>
[fowlks@node1a test]$ chef tags
3 tags.
life (1 posts)
hgtv (1 posts)
fun (1 posts)
</code>

In my cases it appears i only have one post that contains tags.

posts/
posts/2014-06-13_check_me.html
<b>posts/2014-06-16_next_level.html</b>

In my post it contains the contents below.

---

category: life
tags: [life, hgtv, fun]
title: Sample 2 - Nice, Nice and more Nice!
## time: 1:54:56

This is some sample text. It has stuff like
_paragraphs_ and _emphasized words_ and all
that jazz. It's seriously pretty cool,
especially since it's so **natural** to write.

So Basically:
- It's easy
- It's useful
- It creates nice typography
